### PR TITLE
Add Toast notification blog post

### PR DESCRIPTION
**Pull request by asklar**
I saw there was a question about notifications on the RNW repo so I figured I'd write a native module for it, and it seemed like a good idea to write a blog post about it. 

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-windows-samples/pull/450)

### CODE REVIEW

**Comment by asklar** [comment]
cc @qmatteoq

File: website/blog/2021-06-01-notifications.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+Some templates include text only, while others include text and images. Every template has some combination of these. You can see a list of the available templates [here](https://docs.microsoft.com/en-us/previous-versions/windows/apps/hh761494(v=win.10)).
+
+We will create a native module which will have a single method, `raise`. The Windows toast notification API takes an XML document as input, which will specify the text and image sources that we want to show. However, it would be much more natural for us to work with JavaScript objects and JSON instead of XML markup.
+So our `raise` method will take a JS object, which will include what template the app wants to use, as well as arrays of text and images that apply to the template. We will take the information in this JS object, and use it to construct the XML that we need to pass to the platform API.
+
+### Create a module 
+
+For the purpose of our demo, we will add a native module directly to a C++/WinRT example app. If you prefer, you can follow the steps below and apply them to a separate native module project.
```
Comment (stmoy) [inline website/blog/2021-06-01-notifications.md:26]:
```suggestion
For the purpose of our demo, we will add a native module directly to a C++/WinRT example app. If you prefer, you can follow the steps below and apply them to a separate native module project. Note that you can also use C# to write native modules in a similar manner. 
```

**Review by stmoy — APPROVED** [review]
Thank you for writing this blog post 🙏

File: website/blog/2021-06-01-notifications.md (hunk 1 above)
Comment (asklar) [reply website/blog/2021-06-01-notifications.md:26]:
```suggestion
For the purpose of our demo, we will add a native module directly to a C++/WinRT example app. If you prefer, you can follow the steps below and apply them to a separate native module project. Note that you can also use C# to write native modules in a similar manner.  For more information about choosing between C++/WinRT and C#, see [Native code language choice](https://microsoft.github.io/react-native-windows/docs/native-code-language-choice).

**Review by acoates-ms — APPROVED** [review]

**Pull request merged**